OnlineStore/OnlineStore/Filling.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OnlineStore
+{
+    internal class Filling
```

Comment:
No Model / Service separation, as we discussed in class, as well as no folder separation. Please, fix that part